src/index.tsx

```diff
@@ -130,20 +145,17 @@ const observerData: ObserverData = {
         },
         loadTime: null
     },
+    runtime: {
+        deviceType: ''
+    },
     morphs: null,
     spinner: false,
     error: null,
     xrSupported: false,
-    xrActive: false
+    xrActive: false,
```

Comment:
I thought we disallowed trailing commas in object definitions. Is ESLint not running on tsx files in this project?

Reply:
I guess not? @playcanvas/eslint-config/index.js:

<img width="663" alt="Screenshot 2023-10-09 at 11 32 34" src="https://github.com/playcanvas/model-viewer/assets/11276292/d48cc6e3-ce7c-4b32-9870-1adfe7139cc6">